app/ui-react/syndesis/src/modules/extensions/pages/ExtensionDetailsPage.tsx

```diff
@@ -84,7 +84,7 @@ export default class ExtensionDetailsPage extends React.Component {
                         error={error}
                         loading={!hasData}
                         loaderChildren={<PageLoader />}
-                        errorChildren={<ApiError />}
+                        errorChildren={<ApiError error={error as Error} />}
```

Comment:
Actually no, when used with a `WithXyz` component error should be the `errorMessage` property that comes from it. Like [here](https://github.com/syndesisio/syndesis/pull/6008/files#diff-7d7b882bd5054157b0365bad6d545241R156)

Reply:
Thanks! I have added a new commit hopefully fixing things